llvm/lib/CodeGen/GlobalISel/CombinerHelper.cpp

```diff
@@ -5226,6 +5224,8 @@ MachineInstr *CombinerHelper::buildUDivUsingMul(MachineInstr &MI) {
     // TODO: Use undef values for divisor of 1.
     if (!Divisor.isOne()) {
 
+      unsigned KnownLeadingZeros =
```

Comment:
This will get called for every element of a vector, but will always calculate the same value since it's using `LHS` which doesn't change. This is bad for compile time.

Either put it back where it was or count the leading zeros of `C`.

Reply:
Fixed!